src/quick-start.md

```diff
@@ -2,45 +2,59 @@
 
 The Kubewarden stack is made of the following components:
 
-* An arbitrary number of `ClusterAdmissionPolicy` resources: this is how policies
-  are defined inside of Kubernetes
-* An arbitrary number of `PolicyServer` resources, which represent a Deployment of a Kubewarden `policy-server`.
-  This component loads all the policies defined by the administrators and evaluates them
-* A Deployment of `kubewarden-controller`: this is the controller
-  that monitors the `ClusterAdmissionPolicy` resources and interacts
-  with the Kubewarden `policy-server`
+* An arbitrary number of `ClusterAdmissionPolicy` resources: this is how policies are defined inside Kubernetes
+* An arbitrary number of `PolicyServer` resources: this component represents a Deployment of a Kubewarden `policy-server`. The policies defined by the administrators are loaded and evaluated by the Kubewarden `policy-server`
+* A Deployment of `kubewarden-controller`: this is the controller that monitors the `ClusterAdmissionPolicy` resources and interacts with the Kubewarden `policy-server` components
+
+## Installation
+
+> **PREREQUISITES:**
+>
+> Currently, the chart depends on cert-manager. Make sure you have [`cert-manager` installed](https://cert-manager.io/docs/installation/) *before* installing the kubewarden-controller chart.
+>
+> You can install the latest version of `cert-manager` by running the following commands: 
+>
+```shell
```

Comment:
Nit: I think using `shell` instead of `console` for the code block is ok, but note that the lone `wait` gets highlighted then (and other tokens may get too, such as `for`, or so).

Reply:
oh ok, good point, I wanted to have a standardization over the code blocks that are not language (read: YAML) specific.

@viccuad @flavio should I go for `console` instead then?

Reply:
I would prefer `console`, yep :).